### PR TITLE
fix: Teleport Bridge: Rename Statemine to AssetHub Kusama 

### LINKED ISSUE #7155: Teleport Bridge: Rename Statemine to AssetHub Kusama
It would be cool to have unified naming on the platform. Please rename Statemine to Kusama Asset Hub (or AssetHub Kusama or something like that, or keep both names, idk whats best) in Profile - Teleport Bridge. 

## Here:
Screenshot 2023-09-05 at 10 53 01

### It is still called Statemine:
Screenshot 2023-09-05 at 10 59 33

### PR DESCRIPTION
**Pull request by hassnian**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #7155

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; 

CleanShot 2023-09-13 at 15 38 50@2x

### <samp>🤖 Generated by Copilot at fe99e7b</samp>

Refactored the `Teleport.vue` component and added a utility function for chain names. This enhances the code quality and readability of the teleport feature.

### <samp>🤖 Generated by Copilot at fe99e7b</samp>

> _No more hard-coded chains of doom_
> _We use a function to get the name_
> _Importing the prefix from a module of truth_
> _We refactor the code and unleash the flame_

### CODE REVIEW

**Comment by prury** [comment]
@hassnian could you also rename down there? 
image

**Comment by hassnian** [comment]
mb forgot about that :d

CleanShot 2023-09-13 at 18 55 59@2x

**Review by prury — APPROVED** [review]

File: utils/chain.ts [hunk 1]
```diff
@@ -15,6 +15,10 @@ export const getChainNameByPrefix = (prefix: string) => {
   return prefix
 }
 
+export const getChainName = (prefix: Prefix) => {
+  return NAMES[prefix].split(' [Beta]').join('')
```
Comment (Jarsen136) [inline utils/chain.ts:19]: It would be more simple
```suggestion
  return NAMES[prefix].replace(' [Beta]', '')
```

Comment (hassnian) [reply utils/chain.ts]: yep

**Review by Jarsen136 — APPROVED** [review]

**Comment by dudo50** [comment]
Please keep in mind, that the call itself should still contain the name Statemine or Statemint as they are not yet renamed in our XCM SDK. We plan on changing it but the runtime of these chains is still called in the original way which is confusing for everyone. Thus we kept this naming original for now.
Screenshot 2023-09-17 at 13 03 20

**Comment by hassnian** [comment]
> Please keep in mind, that the call itself should still contain the name Statemine or Statemint as they are not yet renamed in our XCM SDK. We plan on changing it but the runtime of these chains is still called in the original way which is confusing for everyone. Thus we kept this naming original for now.  (This does not limit you from renaming labels)
> 
> Screenshot 2023-09-17 at 13 03 20
> 
> 

yep saw that and that has not changed, it still calls Statemine and Statemint

**Review by preschian — APPROVED** [review]

**Review by daiagi — APPROVED** [review]

**Comment by hassnian** [comment]
@yangwao payout missing ty

**Comment by yangwao** [comment]
pay 20 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $20 @ 4.03 USD/DOT ~ 4.963 $DOT
🧗 [13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh](https://polkadot.subscan.io/account/13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)
🔗 [0xf6f1aac0d301fc21b330a1f48c8e9e2ea86d469b8421d3009dcabcfe84dfc338](https://polkadot.subscan.io/extrinsic/0xf6f1aac0d301fc21b330a1f48c8e9e2ea86d469b8421d3009dcabcfe84dfc338)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**